src/components/features/Home/GoodsRankingSection/index.tsx

```diff
@@ -15,19 +15,43 @@ export const GoodsRankingSection = () => {
     rankType: 'MANY_WISH',
   });
 
-  // GoodsMockData를 21번 반복 생성
+  const [goodsList, setGoodsList] = useState<GoodsData[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<string | null>(null);
```

Comment:
`isError` 면 boolean 을 나타낼거 같은데 message 로 지정되어 있네요.

```suggestion
  const [errorMessage, setErrorMessage] = useState<string>('');
```